app/cdap/components/ParsingDrawer/index.tsx

```diff
@@ -0,0 +1,146 @@
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Button } from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
+import { createWorkspace } from 'components/Connections/Browser/GenericBrowser/apiHelpers';
+import { ConnectionsContext } from 'components/Connections/ConnectionsContext';
+import DataPrepStore from 'components/DataPrep/store';
+import DrawerWidget from 'components/DrawerWidget';
+import PositionedSnackbar from 'components/SnackbarComponent/index';
+import T from 'i18n-react';
+import React, { useContext, useEffect, useState } from 'react';
+import ParsingPopupBody from './Components/ParsingPopupBody';
+import {
+  defaultConnectionPayload,
+  defaultErrorOnTransformations,
+  defaultProperties,
+} from './defaultValues';
+import { useStyles } from './styles';
+
+export default function({ setLoading, updateDataTranformation }) {
+  const [drawerStatus, setDrawerStatus] = useState(true);
+  const [properties, setProperties] = useState(defaultProperties);
+  const [schemaValue, setSchemaValue] = useState(null);
+  const { onWorkspaceCreate } = useContext(ConnectionsContext);
+  const [errorOnTransformation, setErrorOnTransformation] = useState(defaultErrorOnTransformations);
+  const [connectionPayload, setConnectionPayload] = useState(defaultConnectionPayload);
+
+  const classes = useStyles();
+  const { dataprep } = DataPrepStore.getState();
+
+  useEffect(() => {
+    setConnectionPayload({
+      path: dataprep.insights.path,
+      connection: dataprep.insights.name,
+      sampleRequest: {
+        properties: {
+          ...properties,
+          schema: schemaValue != null ? JSON.stringify(schemaValue) : null,
+          _pluginName: null,
+        },
+        limit: 1000,
+      },
+    });
+    setDrawerStatus(true);
+  }, [dataprep, properties, schemaValue]);
+
+  const createWorkspaceInternal = async (entity, parseConfig = {}) => {
+    try {
+      setLoading(true);
+      const wid = await createWorkspace({
+        entity,
+        connection: dataprep.insights.name,
+        properties: connectionPayload.sampleRequest.properties,
+      });
+      if (onWorkspaceCreate) {
+        return onWorkspaceCreate(wid);
+      }
+      setDrawerStatus(false);
+      updateDataTranformation(wid);
+    } catch (err) {
+      setErrorOnTransformation({
+        open: true,
+        message: 'Selected Transformation Cannot Be Applied',
+      });
```

Comment:
replace hardcoded data with translation

Reply:
SnackBar:
    errorMessage:
      transformation: 'Selected Transformation Cannot Be Applied'
T.translate(`features.errorMessage.transformation`)
Added to yaml file and changed the interface.